certbot-ci/setup.py

```diff
@@ -0,0 +1,45 @@
+from setuptools import setup
+from setuptools import find_packages
+
+
+version = '0.32.0.dev0'
+
+install_requires = [
+    'pytest',
+    'pytest-cov',
+    'pytest-xdist',
+    'pytest-sugar',
```

Comment:
We'll want to pin a version of this in `tools/dev_constraints.txt`.

Reply:
Done.